src/Batteries.purs

```diff
@@ -1775,6 +1782,74 @@ import Math
   , tan
   , (%)
   )
+import Optic.Core
+  ( (..)
+  )
+import Optic.Getter
+  ( to
+  , view
+  , (^.)
+  )
+import Optic.Laws.Lens
+  ( getSet
+  , setGet
+  , setSet
+  , validLens
+  )
```

Comment:
I don't really think these should be exported, but they don't conflict with anything. I think later on I'll need to go back over the exports and only export things that make sense.